logger.go

```diff
@@ -121,6 +129,22 @@ func (l *Logger) output(s severity, txt string) {
 	}
 }
 
+// Close closes all the underlying log destinations, which will flush all the
+// cached logs. Pleaes call Close once all logging is done. This is commonly
```

Comment:
Please (spelling)

Reply:
I'll run it through a spell checker first ;)